src/components/OcRadio.vue

```diff
@@ -1,17 +1,17 @@
 <template>
-  <label :for="id" :class="{ 'oc-cursor-pointer': !disabled }">
+  <span :class="{ 'oc-cursor-pointer': !disabled }">
```

Comment:
You need to move the cursor class to the label. Having the oc-cursor-pointer class on the span, I don't see a pointer on the label, only on the radiobutton itself. Moving it to the label again works. Not needed on the input, as it already has the pointer by default.

Reply:
IMO it makes sense to move the classes of the label into a computed property then, since there are zero to two classes for the label.